pkgs/top-level/perl-packages.nix

```diff
@@ -2031,6 +2031,20 @@ let self = _self // overrides; _self = with self; {
       maintainers = [ maintainers.rycee ];
     };
   };
+  
+  ConfigIniFiles = buildPerlPackage rec {
+    name = "Config-IniFiles-2.38";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/W/WA/WADG/${name}.tar.gz";
+      sha256 = "0cz5gz5qk729nrbmamz84i3335cc3vfzz8z450crrvzip4isdpgc";
+    };
+#    propagatedBuildInputs = [ MixinLinewise ];
```

Comment:
Why is this commented? Either remove to include or add another comment to explain the rationale.